djangoproject/templates/overview.html

```diff
@@ -1,7 +1,8 @@
 {% extends 'base.html' %}
 
-{% block title %}Django overview{% endblock %}
 {% block layout_class %}full-width{% endblock %}
+{% block title %}Django overview{% endblock %}
+{% block og_title %}Django overview{% endblock %}
```

Comment:
Moving existing code as little as possible helps reviewers
```suggestion
{% block title %}Django overview{% endblock %}
{% block layout_class %}full-width{% endblock %}
{% block og_title %}Django overview{% endblock %}
```